source/inference.md

```diff
@@ -333,33 +283,14 @@ Both the first and last few entries of the resulting data frame are printed
 below to show that we end up with 20,000 point estimates, one for each of the 20,000 samples.
 
 ```{code-cell} ipython3
-:tags: [remove-cell]
-
-# Now that we have obtained the samples, we need to compute the
-# proportion of entire home/apartment listings in each sample.
-# We first group the data by the `replicate` variable&mdash;to group the
-# set of listings in each sample together&mdash;and then use `summarize`
-# to compute the proportion in each sample.
-# We print both the first and last few entries of the resulting data frame
-# below to show that we end up with 20,000 point estimates, one for each of the 20,000 samples.
-```
-
-```{code-cell} ipython3
-# calculate the the number of observations in each sample with room_type == 'Entire home/apt'
 sample_estimates = (
-    samples.query("room_type == 'Entire home/apt'")
-    .groupby("replicate")["room_type"]
-    .count()
-    .reset_index()
-    .rename(columns={"room_type": "counts"})
+    samples
+    .groupby('replicate')
+    ['room_type']
+    .value_counts(normalize=True)
+    .reset_index(name='sample_proportion')
+    .query('room_type=="Entire home/apt"')
 )
-
-# calculate the proportion
-sample_estimates["sample_proportion"] = sample_estimates["counts"] / 40
-
-# drop the count column
-sample_estimates = sample_estimates.drop(columns=["counts"])
-
```

Comment:
While I find that this section is cleaner than it used to be, I wish it could be even simpler. Especially the index collision and need to use `name=` is unfortunate since I don't think we mention that in the wrangling chapter, so we might want to add a sentence if we stick with it here.

Reply:
I couldn't find a good way to do this that didn't include something new such as value counts on  a data frame instead of series, or using `as_index=False` with groupby (both also would include an extra rename step). The current behavior is [listed asa  bug with a pending PR in pandas](https://github.com/pandas-dev/pandas/issues/49497), so in 2.0 it will work without the `name=` in the way we have written it now.